.yarnrc.yml

```diff
@@ -2,6 +2,8 @@ compressionLevel: mixed
 
 enableGlobalCache: false
 
+enableScripts: false
```

Comment:
I know that `@swc/core` has a postinstall script; I was mildly worried that things wouldn't work without that script running, but everything appears to be fine.